update/application.go

```diff
@@ -141,6 +145,16 @@ func (cmd *applicationCmd) applyUpdates(app *apps.Application) {
 	if cmd.Env != nil {
 		buildEnv = *cmd.BuildEnv
 	}
+
+	if cmd.RetryBuild != nil {
+		if *cmd.RetryBuild {
+			if buildEnv == nil {
```

Comment:
This somehow looks a bit too nested :-) I would do a `buildEnv := make(map[string]string)` on line 144 and then you can simplify this:

```go
if cmd.RetryBuild != nil && *cmd.RetryBuild {
  buildEnv[BuildTrigger] = time.Now().UTC().Format(time.RFC3339)
}
```